scripts/publish-root.gradle

```diff
@@ -0,0 +1,38 @@
+// Create variables with empty default values
+ext['signing.keyId'] = ''
+ext['signing.password'] = ''
+ext['signing.key'] = ''
+ext['signing.secretKeyRingFile'] = ''
+ext['ossrhUsername'] = ''
+ext['ossrhPassword'] = ''
+ext['sonatypeStagingProfileId'] = ''
+
+File secretPropsFile = project.rootProject.file('local.properties')
```

Comment:
Check that the 'local.properties' file is not in the GitHub repository.

Reply:
@sravank9 @DemuduSwamy1

Reply:
Double-checked this, the `local.properties ` file is not there in the GitHub repository.